src/types/InfinispanTypes.ts

```diff
@@ -337,17 +337,18 @@ interface CacheFeatureStep {
 }
 
 interface AdvancedConfigurationStep {
-  storage: string | undefined;
-  concurrencyLevel: number | undefined;
-  isolationLevel: string | undefined;
-  lockAcquisitionTimeout: number | undefined;
-  striping: boolean;
+  storage?: string;
+  concurrencyLevel?: number;
+  isolationLevel?: string;
+  lockAcquisitionTimeout?: number;
+  striping?: boolean;
   indexReader: number | undefined;
```

Comment:
aren't these not mandatory too ?

Reply:
We have to pass those as an optional (?) parameter, because in the end if they doesn't have a value it won't show in the Cache Config.